.github/workflows/style-and-sp-check.yml

```
@@ -24,8 +24,30 @@ jobs:
         with:
           ref: ${{ github.event.pull_request.head.ref }}
 
-      - name: Install styler
-        run: Rscript -e "install.packages('styler')"
+      - name: Install packages
+        run: Rscript -e "install.packages(c('styler', 'spelling'))"
+
+      - name: Run spell check
+        id: spell_check_run
+        run: |
+          Rscript components/spell-check.R
+          results=$(Rscript "components/spell-check.R")
```

No need to run the script twice!
```suggestion
          results=$(Rscript "components/spell-check.R")
```